gcs/src/unit_tests/gcs_group_test.cpp

```diff
@@ -552,111 +552,55 @@ START_TEST(test_gcs_group_find_donor)
 #define SARGS(s) s, strlen(s)
     //========== sst ==========
     donor = gcs_group_find_donor(&group, sv, joiner, SARGS("home3"),
-<<<<<<< HEAD
                                  &empty_uuid, GCS_SEQNO_ILL, false);
     fail_if(donor != -EHOSTDOWN);
```

Comment:
wasn't this fail_if changed to ck_assert in upstream?

Reply:
Thanks for catching this. Will fix them in the next revision.